### ISSUE TITLE
Batch button clicker

### ISSUE THREAD

**Issue by andytuba** [honestbleeps/Reddit-Enhancement-Suite#3955]
If a user clicks the same kind of button[1] several times in a short time span and there are many more of that kind of button[2], e.g. a post's "hide" button on a post listing, show a notification or temporary guider next to the button with a CTA: 

> Clippy is wondering:
> You've clicked the {{button.textContent}} button several times recently. 
> Should I __[click all {{remaining.length}} button(s)?]__ [Never offer](/#)

Clicking all the buttons should space out 

[1] Button is an actual `<button>` or a flatlist button (link)

AVOID ABUSE:
* Exclude vote arrows
* Exclude what else?
* Maybe whitelist instead?

[2] For determining "same", perhaps compare "text content is equal" and "class lists intersect exactly except for thing-IDs".  "Many more of that" == >3

**Comment by honestbleeps** [comment]
there are of course API limit concerns on this too.... the UX is going to be rather poor if there's 50 additional buttons that need clicking, and we need to stay 3 seconds apart...

**Comment by andytuba** [comment]
Yeah, that's why I"m thinking of showing a status bar, so people know they should leave that tab alone for a while.
